filter/generic/util.go

```diff
@@ -31,13 +31,15 @@ import (
 // isCallingToGenericService check if it calls to a generic service
 func isCallingToGenericService(invoker protocol.Invoker, invocation protocol.Invocation) bool {
```

Comment:
This case is that a plain invocation will be converted to the generic fashion on the filter, e.g. `a plain invocation -> <filter> -> a generic invocation`. The method name of the plain invocation is neither `$invoke` nor `$invokeAsync`, in other words, `invocation.MethodName() != constant.Generic && invocation.MethodName() != constant.GenericAsync`.